addons/website/static/src/scss/website.scss

```diff
@@ -572,8 +572,8 @@ table.table_desc tr td {
     // Default
     &.s_carousel_default {
         // Controls - chevron
-        .carousel-control-prev-icon:before { content: "\f053" #{"/*rtl:'\f054'*/"}; margin-left: 1.5rem; }
-        .carousel-control-next-icon:before { content: "\f054" #{"/*rtl:'\f053'*/"}; margin-right: 1.5rem; }
+        .carousel-control-prev-icon:before { content: "\f053"; margin-left: 1.5rem; }
+        .carousel-control-next-icon:before { content: "\f054"; margin-right: 1.5rem; }
```

Comment:
Seeing the PR removing explicit RTL tweaks to fix an RTL issue, I would expect:
- A mention of the commit which introduced that, in this case https://github.com/odoo/odoo/commit/ebb61753bf3d3dd8d3f53db088112b9e4beb813d
- A mention of why it can be removed / why it was used

Reply:
@rdeodoo The necessary is made. Could you tell me if it fits your needs?